test/functional/wallet_balance.py

```diff
@@ -0,0 +1,133 @@
+#!/usr/bin/env python3
+# Copyright (c) 2014-2018 The Bitcoin Core developers
```

Comment:
2018? Or is this based on existing stuff?

Reply:
no, this should be 2018